lib/ap/internal/sys9.h

```diff
@@ -91,13 +89,13 @@ enum
 // Remove unused functions now from libc
 
 extern	int	await(char*, int);
-//extern	int	alarm(unsigned long);
+//extern	int	alarm(uint32_t);
 #ifndef __SYS_SOCKET_H__
 extern	int	bind(const char*, const char*, int);
 #endif
-extern	int	chdir(const char*);
+extern	int	_CHDIR(const char*);
```

Comment:
It's not for that, it's due to chdir.c is based on chdir syscall. The alternative is drop ap/plan9/chdir.c at all.